### DISCUSSION TITLE
How to disable contextMenu?

### DISCUSSION THREAD

**Discussion by EGo14T** [wailsapp/wails#2041] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/28916447/199639181-93feac13-4f0f-4ef2-904a-5e85e4b3a0b2.png)

This JavaScript only works on the <body/>
```html
<body oncontextmenu="return false;"></body>
```
However, it does not take effect outside the body area

**Comment by EGo14T** [comment, marked as answer]
it works in `<html/>` 😂

**Comment by alanchenchen** [comment]
In production mode, contextMenu is disabled by default
